README.md

```diff
@@ -30,15 +30,28 @@ Create a virtualenv if required and install required packages using "pip install
     1. Brief Information: The CI_DailyBuildUpdates.py script when invoked with command line arguement info_type as "brief" it will display Build type, job id, cluster deployment status, Lease and total number of failed testcases.
 
         ```python3 CI_DailyBuildUpdates.py --info_type brief```
+        
+    2. The CI_DailyBuildUpdates.py script when invoked with command line arguements info_type as "brief" and zone, it will display the builds details in the provided zone.
+        
+        ```python3 CI_DailyBuildUpdates.py --info_type brief --zone syd04```
 
-    2. Detailed Information: The CI_DailyBuildUpdates.py script when invoked with command line arguement info_type as "detailed" it will display the Job id, job link, cluster deployment error message if it occured, Nightly image used, Node status, Checks for Node crash and lists all the failed testcases.  
+    3. Detailed Information: The CI_DailyBuildUpdates.py script when invoked with command line arguement info_type as "detailed" it will display the Job id, job link, cluster deployment error message if it occured, Nightly image used, Node status, Checks for Node crash and lists all the failed testcases.  
 
         ```python3 CI_DailyBuildUpdates.py --info_type detailed```
 
+    4. The CI_DailyBuildUpdates.py script when invoked with command line arguements info_type as "detailed" and zone, it will display the builds details in the provided zone.
+
+        ```python3 CI_DailyBuildUpdates.py --info_type detailed --zone syd04```
+
 
 
 2. **CI_JobHistory.py:** The CI_JobHistory.py is a interactive script which allows user to query a specific information from all builds that ran on the CI system within a given date range.  
+    
     ```python3 CI_JobHistory.py```
 
+    1. The CI_JobHistory.py when invoked with command line arguement zone, it will get all builds that ran in the particular zone.
```

Comment:
Please fix the typo "argument" in this file

Reply:
Done